inlong-sort-standalone/sort-standalone-source/src/main/java/org/apache/inlong/sort/standalone/dispatch/DispatchManager.java

```diff
@@ -134,6 +143,18 @@ public void outputOvertimeData() {
                 inCounter.getAndSet(0), outCounter.getAndSet(0));
     }
 
+    /**
+     * reset dispatch time if the dispatch time is invalid.
+     * The default ahead time is 1 hour, and default delay time is 16 hours.
+     */
+    private void checkAndResetDispatchTime(DispatchProfile dispatchProfile) {
+        long cur = System.currentTimeMillis();
```

Comment:
The code line can be before the line "removeKeys.forEach((key) -> {".

long cur = System.currentTimeMillis();